quickstart-sda.md

```diff
@@ -0,0 +1,239 @@
+# Quickstart guide to the SDA
+
+This document contains information on how to work with the Sensitive Data Archive (SDA) from the GDI user perspective.
+
+## Brief description of the storage and interfaces stack
+
+The storage and interfaces software stack for the GDI-starter-kit consists of the following services:
+
+| Component     | Description |
+|---------------|------|
+| broker        | RabbitMQ based message broker, [SDA-MQ](https://github.com/neicnordic/sda-mq). |
+| database      | Postgres database, [SDA-DB](https://github.com/neicnordic/sda-db). |
```

Comment:
Even though "Postgres" is [an accepted nickname for PostgreSQL](https://wiki.postgresql.org/wiki/FAQ#What_is_PostgreSQL.3F_How_is_it_pronounced.3F_What_is_Postgres.3F), we should use the full name.

```suggestion
| database      | PostgreSQL database, [SDA-DB](https://github.com/neicnordic/sda-db). |
```